Wbooks/src/app/screens/library/components/bookList/styles.js

```diff
@@ -0,0 +1,14 @@
+import {StyleSheet} from 'react-native';
+
+const styles = StyleSheet.create({
+   container: {
+    display:'flex',
+    backgroundColor:'#EAF6FA',
```

Comment:
Move color to const color file, import and replace them.

Reply:
done